packages/core/test/functions.spec.ts

```diff
@@ -644,405 +520,6 @@ describe('Stylable functions (native, formatter and variable)', () => {
-                expectTransformDiagnostics(
-                    {
-                        entry: '/style.st.css',
-                        files: {
-                            '/style.st.css': {
-                                content: `
-                            :vars {
-                                myVar: st-map(
-                                    key1 red,
-                                    key2 st-map(
-                                        key3 green
-                                    )
-                                );   
-                            }
-                            .root {
-                                color: value(myVar, key1);
-                                color: value(myVar, key2, key3);
-                                |color: value(myVar, key2, key4)|;
-                            }
-                            `,
-                            },
-                        },
-                    },
-                    [
-                        {
-                            message: functionWarnings.COULD_NOT_RESOLVE_VALUE('myVar, key2, key4'),
-                            file: '/style.st.css',
-                        },
-                    ]
-                );
-            });
-
-            it('should return multiple warnings for unknown custom-value keys and too many arguments in a simple var', () => {
-                expectTransformDiagnostics(
-                    {
-                        entry: '/style.st.css',
-                        files: {
-                            '/style.st.css': {
-                                content: `
-                            :vars {
-                                v1: red;
-                                v2: green;
-                                myVar: st-map(
-                                    key1 red,
-                                    key2 st-map(
-                                        key3 green
-                                    )
-                                );
-                            }
-                            .root {
-                                |background: value(myVar, key2, key4), value(v1, v2)|;
-                            }
-                            `,
-                            },
-                        },
-                    },
-                    [
-                        {
-                            message: functionWarnings.COULD_NOT_RESOLVE_VALUE('myVar, key2, key4'),
-                            file: '/style.st.css',
-                        },
-                        {
-                            message: functionWarnings.MULTI_ARGS_IN_VALUE('v1, v2'),
-                            file: '/style.st.css',
-                        },
-                    ]
-                );
-            });
-
-            it('should return a warning when passing more than one argument to custom value with working fallback', () => {
```

Comment:
check that the fallback still works and tested

Reply:
added case to tests